### ISSUE TITLE
[types/convert-typeof-to-is-type]: Function insertion doubles the class comment.

### ISSUE THREAD

**Issue by ElPrudi** [coderaiser/putout#164]
Every time this rule is triggered and inserts a function into the file where a class is, the class comment gets duplicated.

For example: `X.ts`:

```ts
/**
 * This is class X
 */
export default class X {

   public a(b): string {
          if (typeof b === 'number') return 'doStuff'
          return 'stuff'
   }
}
```

gets transformed into

```ts
/**
 * This is class X
 */
const isNumber = (x) => typeof x === 'number'

/**
 * This is class X
 */
export default class X {

   public a(b): string {
          if (isNumber(x)) return 'doStuff'
          return 'stuff'
   }
}

**Comment by coderaiser** [comment]
Could you please show me your `.putout.json`?

**Comment by ElPrudi** [comment]
I use `eslint-plugin-putout` and this is the putout part of it:

```json
{
	"putout/putout": [
		2,
		{
			"plugins": ["apply-shorthand-properties", "conditions", "simplify-ternary", "types"],
			"rules": {
				"apply-shorthand-properties": [
					"on",
					{
						"ignore": []
					}
				],
				"split-variable-declarations": "off",
				"new/remove-useless": "off",
				"remove-unused-variables": "off",
				"remove-console": "off",
				"convert-template-to-string": "off",
				"remove-useless-array-constructor": "off",
				"types/remove-double-negations": "off",
				"promises/remove-useless-async": "off",
				"convert-object-assign-to-merge-spread": "off",
				"apply-destructuring/array": "off",
				"apply-destructuring/object": "off",
				"promises/add-missing-await": "off",
				"typescript/remove-useless-parens": "off",
				"remove-useless-arguments/method": "off",
				"math/apply-exponentiation": ["on", {
					"exclude": [
						"__a * __a",
						"__a * __a * __a",
						"__a * __a * __a * __a",
						"__a * __a * __a * __a * __a",
					]
				}]
			}
		}
	]
}

```

**Comment by coderaiser** [comment]
I can't reproduce this with `1.ts`:

```js
/**
 * This is class X
 */
export default class X {

   public a(b): string {
          if (typeof b === 'number') return 'doStuff'
          return 'stuff'
   }
}
```

```sh
putout 1.ts --fix
```

Produces this result:

```js
const isNumber = (a) => typeof a === 'number';

/**
 * This is class X
 */
export default class X {
    public a(b): string {
        if (isNumber(b))
            return 'doStuff';

        return 'stuff';
    }
}
```

**Comment by ElPrudi** [comment]
Found it. It only duplicates it, if you also have imports before that.

**Comment by coderaiser** [comment]
Fixed in [`v30.6.0`](https://github.com/coderaiser/putout/releases/tag/v30.5.0). Please re-install 🐊.

Is it works for you?

**Comment by ElPrudi** [comment]
Yes, it works now. Thank you very much :)
